p-token/src/processor/withdraw_excess_lamports.rs

```diff
@@ -0,0 +1,80 @@
+use pinocchio::{
+    account_info::AccountInfo,
+    program_error::ProgramError,
+    sysvars::{rent::Rent, Sysvar},
+    ProgramResult,
+};
+use spl_token_interface::{
+    error::TokenError,
+    state::{account::Account, load, mint::Mint, multisig::Multisig, Transmutable},
+};
+
+use super::validate_owner;
+
+#[inline(always)]
```

Comment:
Totally separate from this PR, but what's the impact of these inlines? I wonder if there's a chance that we're actually making things worse by adding this annotation everywhere.

Reply:
In some cases it helps a lot – e.g., for `transfer`, inlining brings the CUs from `227` to `143`.

But for others there is no difference in terms of CU, e.g., `withdraw_excess_lamports`, although it increases the program binary (.

I can revise all the inlines in a separate PR and only keep the ones that make a difference on the CUs. This way we can optimize both CU and program size.

Reply:
Ah interesting, I'm glad to see that it does help in some cases!